### PR TITLE
docs: add `RadioGroup` documentation

### PR DESCRIPTION
**Pull request by plagoa**
Some changes to the `Playground` to allow receiving children wrapped in a React Fragment.

### CODE REVIEW

File: apps/docs/src/pages/components/radio-group.mdx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    },
+  }}
+  children={
+    <>
+      <HvRadio label="Bar Chart" value="1" />
+      <HvRadio label="Line Chart" value="2" />
+      <HvRadio label="Pie Chart" value="3" />
+    </>
+  }
+/>
```
Comment (zettca) [inline apps/docs/src/pages/components/radio-group.mdx:51]: we should be able to pass `children` normally (without needing the `Fragment`), but there's something wrong with it

ie.
```suggestion
>
  <HvRadio label="Bar Chart" value="1" />
  <HvRadio label="Line Chart" value="2" />
  <HvRadio label="Pie Chart" value="3" />
</Playground>
```

Comment (zettca) [reply apps/docs/src/pages/components/radio-group.mdx:51]: https://github.com/lumada-design/hv-uikit-react/pull/4469

Comment (plagoa) [reply apps/docs/src/pages/components/radio-group.mdx:51]: I'll wait for that to be merged and update this.

Comment (zettca) [reply apps/docs/src/pages/components/radio-group.mdx:51]: I'll wait for your review there! 😆

Comment (plagoa) [reply apps/docs/src/pages/components/radio-group.mdx]: Updated.

**Review by francisco-guilherme — APPROVED** [review]

**Pull request merged**